docs/contribute.md

```diff
@@ -0,0 +1,55 @@
+# Contributor guide
+
+These sections describe how you can make contributions to this theme.
+
+
+## Run the tests
+
+The easiest way to run the test suite is using `nox`.
```

Comment:
```suggestion
The easiest way to run the test suite is using [`nox`](https://nox.readthedocs.io/).
```
Nox isn't as well known as the typical Python testing tools, so helpful to include a link.